### DISCUSSION TITLE
希望能添加 showresult 参数的开关

### DISCUSSION THREAD

**Discussion by JiunnTarn** [dreamncn/Qianji_auto#159] [奇思妙想]
钱迹更新了 3.2.1.10 版本，支持不显示记账结果的 SnackBar，只需要在记账 url 后后面加上 `&showresult=0` 即可。

希望大佬能在设置里加一个针对这个参数的开关，谢谢！

**Comment by dreamncn** [comment]
好的

**Reply by JiunnTarn** [reply]
谢谢大佬，新年快乐！
